examples/demo-apps/android/MiniBench/.gitignore

```diff
@@ -0,0 +1,16 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+.cxx
+local.properties
+*.aar
```

Comment:
Is `extension/android/benchmark` or  `extension/benchmark/android` a better place for the new app? It's not a demo for OSS users, so examples/demo-apps won't be a suitable place IMO